static/js/components/CommentForms.js

```diff
@@ -217,7 +226,7 @@ export const ReplyToCommentForm = connect(mapStateToProps, mapDispatchToProps)(
 
     onSubmit = async (event: Event) => {
       const { formKey, forms, post, onSubmit } = this.props
-      const { post_id, text, comment_id } = R.prop(formKey, forms).value
+      const { post_id, text, comment_id } = R.prop(formKey, forms).value // eslint-disable-line camelcase
```

Comment:
What was the camelcase lint error? For comment_id and post_id?

Reply:
yeah, I'm not exactly sure why it wasn't an error before :woman_shrugging: